recipes/r-rocs/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = '1.3' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rocs
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ROCS_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ROCS/ROCS_{{ version }}.tar.gz
+  sha256: 71e647d0909d38a36c8b590e19e736cce1f33694a9cfc38ef2e6aa000e726d87
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```

Comment:
Anything speaking against marking this `noarch: generic`?

Reply:
I thought this parameter can't be selected when building packages for both windows and linux. I may have misunderstood `noarch: generic' option, would love when https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-generic guidlines get written.

Reply:
That was a stupid question on my part, it shouldn't be `noarch` then I think as you had it before.

Reply:
No problem, thanks for reviewing!